### DISCUSSION TITLE
Options for using Libre2 with XDrip+

### DISCUSSION THREAD

**Discussion by simonpickering** [Navid200/xDrip#22] [Libre]
I've read through the "issue" on the XDrip+ github page talking about patching the official LibreView app (https://github.com/NightscoutFoundation/xDrip/issues/1394) but I also see mention of direct comms ability having been merged recently (https://github.com/NightscoutFoundation/xDrip/pull/1524).

What is the recommendation atm as I'm about to move to libre2? 

As far as I can see the options are:

* Use mm2 as per libre1
* Patch the official LibreView app (German version 2.3) and this can talk to XDrip+
* Potential to use libre2 directly with XDrip+ using Tzachi's patches which are now in the mainline code

I think it would be useful to setup (or if there is already one, publicise more widley) a FAQ which is kept up to date would be useful for newcomers, and even those who've been using XDrip+ for a while (though only with libre1 + mm2). 

Trawling through the "issues" which have almost become chats, as well as Facebook and gitter is time consuming and I'd guess probably means people repeatedly ask the same questions.

I'd be more than happy to help with the maintenance/curation of a FAQ wherever it resides.

Thanks,

Simon

_Edit: It would appear this may hold the required answers!_
_https://www.minimallooper.com/post/how-to-setup-freestyle-libre-2-and-oop2-to-use-a-native-bluetooth-connection-in-xdrip_

**Comment by tzachi-dar** [comment]
Seems you have found the answer yourself...

**Reply by simonpickering** [reply]
yes, and thank you for your work with the code to make it happen. 

It did take some trawling though and I still feel that a FAQ would be a good thing. Teleriddler's article (www.minimallooper.com) is very good, but being centralised somewhere would make life easier for people to find the relevant info.
